bukkit/src/main/java/net/william278/huskhomes/BukkitHuskHomes.java

```diff
@@ -174,9 +174,12 @@ public void onEnable() {
 
             // Register plugin hooks (Economy, Maps, Plan)
             this.pluginHooks = new HashSet<>();
-            if (settings.economy && Bukkit.getPluginManager().getPlugin("Vault") != null) {
-                pluginHooks.add(new VaultEconomyHook(this));
-            }
+            if(settings.economy)
+                if (Bukkit.getPluginManager().getPlugin("RedisEconomy") != null) {
+                    pluginHooks.add(new RedisEconomyHook(this));
+                }else if (Bukkit.getPluginManager().getPlugin("Vault") != null) {
+                    pluginHooks.add(new VaultEconomyHook(this));
+                }
```

Comment:
Style: Using standard IntelliJ code style scheme here -- spaces after closing braces and before if statement blocks
e.g. `if (settings.economy) {`, `} else if ()`
You should be able to use the Alt+Enter auto-styling to fix this.

Reply:
Done